src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -548,8 +548,8 @@ public String getValidityInfoForAllowedName(String fieldName, int maxLength, Str
                                             fieldName, REASON_TOO_LONG, maxLength);
         }
         if (!Character.isLetterOrDigit(value.codePointAt(0))) {
-            boolean isStartWithBraces = value.charAt(0) == '{' && value.contains("}");
-            if (!isStartWithBraces) {
+            boolean isBegunWithBraces = value.charAt(0) == '{' && value.contains("}");
```

Comment:
This still feels a little weird to it, but I cant think of any replacement for it.